package.json

```diff
@@ -19,14 +19,16 @@
     "prebuild:clean": "npx expo prebuild --clean --no-install",
     "prebuild": "npx expo prebuild --no-install",
     "update": "node scripts/push-update.js",
-    "select-images": "node scripts/select-images.js"
+    "select-images": "node scripts/select-images.js",
+    "verify-assets:android": "npx expo export --dump-assetmap -p android; npx expo-updates assets:verify --build-manifest-path android/app/build/intermediates/assets/release/app.manifest --exported-manifest-path dist/metadata.json --asset-map-path dist/assetmap.json --platform android",
+    "verify-assets:ios": "npx expo export --dump-assetmap -p ios; npx expo-updates assets:verify --build-manifest-path ios/build/Build/Products/Release-iphonesimulator/EXUpdates/EXUpdates.bundle/app.manifest --exported-manifest-path dist/metadata.json --asset-map-path dist/assetmap.json --platform ios"
```

Comment:
ditto about maybe doing a xcbuild or something (`yarn ios`?) as part of this commit to ensure the intermediates are there